MovieApp/SearchResultsCollectionView.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  SearchResultsCollectionView.swift
+//  MovieApp
+//
+//  Created by Five on 05.05.2022..
+//
+
+
+import Foundation
+import SnapKit
+import UIKit
+import MovieAppData
+
+class SearchResultsCollectionView: UIView {
```

Comment:
Again, I would name this SearchResultsView, since it is a search results view, that contains a collection view, but it itself is not a view. It would be less confusing for someone reading a code.